plugins/mcreator-localization/lang/texts.properties

```diff
@@ -2541,7 +2540,6 @@ elementgui.ranged_item.swinged_by_entity=When entity swings item
 elementgui.ranged_item.can_use=Can use ranged item
 elementgui.ranged_item.make_glow=Make item glow
 elementgui.ranged_item.texture=Texture
-elementgui.ranged_item.enable_glowing=Enable glowing effect
```

Comment:
The translation lines that you removed should also be removed in the other files of the other languages

Reply:
They are automatically handled by Crowdin.

Reply:
Ah, Ok